### PR TITLE
Set up associations

### PR DESCRIPTION
**Pull request by moise-mulungu**
On this milestone of the project, I:
- Created a class named `Classroom`
- Set an instance variable and used `setter` and `getter` for the `CLassroom` class
- Created the `has-many/belongs-to` relationship between `Classroom` and `Student`
- Created a class `Book` with its instance variables
- Created a class `Rental` with `setter` and `getter` to its instance variable
- Create the `many-to-many` relationship between `Person` and `Book` using the intermediate class `Rental`
- Updated the README file.

### CODE REVIEW

File: student.rb [hunk 1]
```diff
@@ -1,15 +1,14 @@
 require './person'
 
 class Student < Person
-  def initialize(age, classroom, name = 'Unknown', parent_permission: true)
+  attr_reader :classroom
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
     super(age, name, parent_permission)
-    @classroom = classroom
+    @classroom = nil
```
Comment (DeliceLydia) [inline student.rb]: - [ ] Please work on the method that serves as `belongs_to` side it mean(student belongs to a classroom)

File: person.rb [hunk 2]
```diff
@@ -22,16 +25,14 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rental.push(rental)
+    rental.person = self
+  end
```
Comment (DeliceLydia) [inline person.rb]: - [ ] Refer to how you are creating a rental in the `book` class here you are supposed to send `book` and `date` as parameters, and the person itself should be sent as a parameter you can do:
```ruby
def add_rental(book, date)
    Rental.new(date, book, self)
  end
```

File: rental.rb [hunk 3]
```diff
@@ -0,0 +1,14 @@
+class Rental
+  attr_reader :book, :person
```
Comment (DeliceLydia) [inline rental.rb]: - [ ] I suggest that `book` and `person` be attribute accessor as well to be accessed and modified at the same time

**Review by DeliceLydia — CHANGES_REQUESTED** [review]
## Hi @moise-mulungu, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! Highlights for your project:
- Followed git workflow
- Good README


## Required Changes ♻️
_Check the comments under the review._



### Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

**Review by Mre55 — APPROVED** [review]
**Hi @moise-mulungu,**

# Status: APPROVED 🏆 🎉

Your project is complete!  There is nothing else to say other than... it's time to merge it  🚀: 
Congratulations! 🎉 

## To highlight 🎯

- The Pull Request is following the GitHub Flow ✅ 
- The PR title and description are professional and descriptive ✅ 
- The README is well documented ✅ 
- Linters are working properly ✅ 




Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**